buf.yaml

```diff
@@ -19,7 +19,7 @@ lint:
       - controller/api/view/v1/view.proto
     FIELD_LOWER_SNAKE_CASE:
       - controller/api/v1/error.proto
-      - controller/custom_options/v1/testing.proto
+      - testing/options/v1/mask_mapping.proto
```

Comment:
Is this change related to metrics? I think this is causing a log of generated files to change and it is not clear if it belongs in this PR.

Reply:
This change is related in that we are adding testing related proto files for options.  I'm adding new testing related proto files for options and this change makes us more in line with the standard that places it under thest /testing/options/.. path.

Reply:
can this change, and the change to the generated files be done in a separate commit (and/or PR)? so that it is easier to see the changes that are directly relevant for the metrics?

Reply:
This was due to an old version of gofumpt.  I've reverted all the reformatted files.   PTAL.